worker/codalabworker/local_run/docker_image_manager.py

```diff
@@ -144,7 +144,12 @@ def get(self, image_spec):
             image_spec += ':latest'
         try:
             image = self._docker.images.get(image_spec)
-            digest = image.attrs.get('RepoDigests', [image_spec])[0]
+            digests = image.attrs.get('RepoDigests', [image_spec])
+            if len(digests) == 0:
+                return ImageAvailabilityState(
+                    digest=None, stage=DependencyStage.FAILED, message='No digest available for {}, probably because it was built locally; delete the docker image on the worker and try again'.format(image_spec),
```

Comment:
```suggestion
                    digest=None, stage=DependencyStage.FAILED, message='No digest available for {}, probably because it was built locally; delete the Docker image on the worker and try again'.format(image_spec),
```